tests/integration/functional/cli/test_utils.py

```diff
@@ -26,10 +28,75 @@ def test_temporarily_setting_the_active_stack():
     }
     new_stack = Client().create_stack(name="new", components=components)
 
-    with temporary_active_stack():
+    with cli_utils.temporary_active_stack():
         assert Client().active_stack_model == initial_stack
 
-    with temporary_active_stack(stack_name_or_id=new_stack.id):
+    with cli_utils.temporary_active_stack(stack_name_or_id=new_stack.id):
         assert Client().active_stack_model == new_stack
 
     assert Client().active_stack_model == initial_stack
+
+
+def test_error_raises_exception():
+    """Tests that the error method raises an exception."""
+    with pytest.raises(Exception):
+        cli_utils.error()
+
+
+def test_file_expansion_works(tmp_path):
+    """Tests that we can get the contents of a file."""
+    sample_text_value = "aria, blupus and axl are the best friends ever"
+    not_from_file_value = "this is not from a file"
+    file_path = tmp_path / "test.txt"
+    file_path.write_text(sample_text_value)
+
+    # test that the file contents are returned
+    assert (
+        cli_utils.expand_argument_value_from_file(
+            name="sample_text", value=f"@{file_path}"
+        )
+        == sample_text_value
+    )
+
+    assert (
```

Comment:
do we ever want to test a non-existing file?

Reply:
Tested now.